config/storybook/webpack.config.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
+const webpack = require('webpack');
+const builds = require('../builds');
+const flatten = require('lodash/flatten');
+const find = require('lodash/find');
+const webpackMerge = require('webpack-merge');
+
+const srcPaths = flatten(builds.map(build => build.paths.src));
+
+const webpackConfigs = require('../webpack/webpack.config');
+const clientWebpackConfig = find(
+  webpackConfigs,
+  config => config.name === 'client'
+);
+
+module.exports = webpackMerge(
+  {
+    // We don't want to apply the entire webpack config,
+    // mainly because it configures entries and outputs,
+    // which would break the Storybook build.
+    module: clientWebpackConfig.module,
+    plugins: clientWebpackConfig.plugins
+  },
+  {
+    plugins: [
+      new webpack.DefinePlugin({
```

Comment:
Does this work because `DefinePlugin` replaces the `__SKU_SRC_PATHS*` tokens before webpack examines the `require` expressions?

Reply:
Yep, exactly.

Reply:
Learned a thing 👍